ui/src/frontend/viewer_page.ts

```diff
@@ -287,8 +287,8 @@ class TraceViewer implements m.ClassComponent<TraceViewerAttrs> {
     );
 
     const overviewPanel = [];
-    if (OVERVIEW_PANEL_FLAG.get()) {
-      overviewPanel.push(m(OverviewTimelinePanel, {key: 'overview'}));
+    if (OVERVIEW_PANEL_FLAG.get() && !globals.hideOverview) {
+        overviewPanel.push(m(OverviewTimelinePanel, {key: 'overview'}));
```

Comment:
Is there a way to set the feature flag from the Sokatoa side instead of also checking for a global?